person.rb

```diff
@@ -0,0 +1,26 @@
+# class person
+class Person
+  # we set the getter and setter for these elements - id, name and age
+  attr_accessor :id, :name, :age
```

Comment:
- [ ] From the project requirements, `id` should only be able to be retrieved so only the Getter is defined
- Kindly make this change.

### Instructions:
![image](https://github.com/benja27/OOP-school-library/assets/50990925/0e0bac59-40cf-498d-a55c-354ea3c10ce2)

Reply:
done!